File: Bridge/Resources/.generated/system/security/accessControl/eventWaitHandleRights.js
    Bridge.define("System.Security.AccessControl.EventWaitHandleRights", {
        $kind: "enum",
        statics: {
            fields: {
                Modify: 2,
                Delete: 65536,
                ReadPermissions: 131072,
                ChangePermissions: 262144,
                TakeOwnership: 524288,
                Synchronize: 1048576,
                FullControl: 2031619
            }
        }
    });

File: Bridge/System/Security/AccessControl/EventWaitHandleRights.cs
using Bridge;

namespace System.Security.AccessControl
{

    public enum EventWaitHandleRights
    {
        Modify = 0x000002,
        Delete = 0x010000,
        ReadPermissions = 0x020000,
        ChangePermissions = 0x040000,
        TakeOwnership = 0x080000,
        Synchronize = 0x100000,
        FullControl = 0x1F0003
    }
}